man/systemd.mount.xml

```diff
@@ -148,16 +148,12 @@
   <refsect1>
     <title><filename>fstab</filename></title>
 
-    <para>Mount units may either be configured via unit files, or via
-    <filename>/etc/fstab</filename> (see
-    <citerefentry project='man-pages'><refentrytitle>fstab</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for details). Mounts listed in <filename>/etc/fstab</filename>
-    will be converted into native units dynamically at boot and when
-    the configuration of the system manager is reloaded. In general,
-    configuring mount points through <filename>/etc/fstab</filename>
-    is the preferred approach. See
```

Comment:
oh, sorry. sure.
Well, `/etc/fstab` just works. `*.mount`-units require some [hacks](https://github.com/systemd/systemd/pull/3152#issuecomment-216025727).
I guess we should document all these hacks first. And drop
> In general, configuring mount points through /etc/fstab is the preferred approach

second